tempel.el

```diff
@@ -379,18 +379,18 @@ If a field was added, return it."
         (push range (car st))
         (overlay-put range 'modification-hooks (list #'tempel--range-modified))
         (overlay-put range 'tempel--range st)
+        (overlay-put range 'tempel--post-callback (plist-get plist :post))
```

Comment:
Can we call this just `tempel--post`? I prefer shorter names if sufficiently unique.

Reply:
Sure; updated!